packages/components/src/FormField/FormField.tsx

```diff
@@ -142,180 +147,179 @@ export interface FormFieldProps {
    * Callback to get the the status and message when validating a field
    * @param messages
    */
-  onValidation?(messages: ValidationProps[]): void;
+  onValidation?(messages?: ValidationProps[]): void;
```

Comment:
Why did you make `messages` optional?

Reply:
Because sometimes there's none

Reply:
I think we should just send an empty array in that case…